<?xml version="1.0"?>

<launch>
  <node name="joy_node" pkg="joy" type="joy_node">
    <rosparam param="dev">/dev/input/js0</rosparam>
  </node>
  <node name="driving_widget" pkg="thor_mang_driving_widget" type="driving_widget" output="screen">
    <rosparam param="camera_topic">/head_cam/image_raw</rosparam>
    <rosparam param="head_controller_topic">/thor_mang/head_traj_controller/command</rosparam>
  </node>
</launch>
